sparsecoding/inference.py

```diff
@@ -269,4 +269,112 @@ def infer(self, data, dictionary):
             
             self.checknan(a,'coefficients')
         return a
+    
+    
+    
+class LSM(InferenceMethod):
+    """
+    Infer coefficients for each image in data using elements dictionary.
+    Method implemented according to "Group Sparse Coding with a Laplacian Scale Mixture Prior" (P. J. Garrigues & B. A. Olshausen, 2010)
+    """    
+
+    def __init__(self, n_iter = 100, n_iter_LSM = 6, BETA=0.01, ALPHA=80.0, sigma=0.005, sparseThreshold = 10**-2, solver = None):
```

Comment:
make all variable names lower case and snake case

camel -> snake
sparseThreshold -> sparse_threshold

Reply:
Modified the names accordingly